app/models/ability.rb

```diff
@@ -0,0 +1,38 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+
+    if user.is? :admin
+      can :manage, :all, Post
+    else
```

Comment:
- [ ] Kindly add a rule that allows authors to delete their posts or comments.

Reply:
Implemented 👍